tasks/20240223-composable-stable-pool-v6/readme.md

```diff
@@ -0,0 +1,8 @@
+# 2024-02-23 - Composable Stable Pool V6
+
+Deployment of `ComposableStablePoolFactory`, which supersedes `20230711-composable-stable-pool-v5`.
```

Comment:
```suggestion
Deployment of `ComposableStablePoolFactory` V6, which supersedes `20230711-composable-stable-pool-v5`.
```
Not really important, but could name it explicitly in the comment, for consistency with the V5 deprecation comment.